doc/source/data/getting-started.rst

```diff
@@ -233,125 +233,3 @@ This is a common pattern useful for loading and splitting data between distribut
 
     ray.get([w.train.remote(s) for w, s in zip(workers, shards)])
     # -> [650, 650, ...]
-
-
-.. _dataset_pipelines_quick_start:
```

Comment:
Where is the content moving?

Reply:
(does it need to be added to front of datasets pipeline page)?

Reply:
Here: https://docs.ray.io/en/master/data/pipelining-compute.html#